_tests/browser_tests/spec_suites/shared/shared_the-bureau.js

```diff
@@ -0,0 +1,19 @@
+TheBureauPage = require( '../../page_objects/page_the-bureau.js' );
```

Comment:
Is `page_` prefix necessary? Will any other type of js file appear in the `/page_objects/ directory?